deploy-jenkins/templates/README.md

```diff
@@ -0,0 +1,154 @@
+# OpenShift 3 Jenkins Templates
+
+## Running Digger on existing OpenShift instance
+
+
+#### Prerequisites:
+- Dedicated OpenShift project (namespace) for jenkins and created build slaves
+- Persistent volume (by default 10Gi, but it can be changed for different needs)
+- Android SDK container (check [docker folder](../docker) for building instructions)
+
+#### Android SDK Steps
```

Comment:
These steps are now ecompassed by the installer so do we need to reference them? See my [pr](https://github.com/aerogear/digger-installer/pull/13) for suggest alternative to this.